include/myst/kernel.h

```diff
@@ -152,6 +152,8 @@ typedef struct myst_kernel_args
     // CPUs reported by sched_getaffinity().
     size_t max_affinity_cpus;
 
+    uint8_t config_id[64];
```

Comment:
When doing remote attestation, mystikos will send
`init_time_claim | base_claim` to the remote service.
`init_time_claim` should contain the hash (which can be obtained from this I assume) that host passes in with `myst_enter_ecall` and the `base_claim` includes the SGX report, which contains the `config_id` field.
We might want the hash in the `init_time_claim` have different naming.